src/main/java/com/google/sps/data/Session.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.Optional;
+import com.google.sps.data.EntityConstants;
+
+/** Class that represents a session. */
+public class Session implements SessionInterface {
```

Comment:
Also, the JavaScript Session class has List of names as one of the columns but it is missing here. Is that something we are adding to the Java class?

Reply:
I'll add it to the Java class.